From 83d00364a3e9e0c1a7b3d51cc9c282a5585036d9 Mon Sep 17 00:00:00 2001
From: "Dmitry V. Levin" <ldv@altlinux.org>
Date: Sat, 18 Aug 2018 07:22:47 +0000
Subject: [PATCH 2/2] xlat: update V4L2_CID_USER_*_BASE constants

* xlat/v4l2_control_id_bases.in (V4L2_CID_USER_S2255_BASE,
V4L2_CID_USER_SI476X_BASE, V4L2_CID_USER_SAA7134_BASE,
V4L2_CID_USER_ADV7180_BASE, V4L2_CID_USER_TC358743_BASE,
V4L2_CID_USER_MAX217X_BASE): New constants.
---
 xlat/v4l2_control_id_bases.in | 14 ++++++++++----
 1 file changed, 10 insertions(+), 4 deletions(-)

diff --git a/xlat/v4l2_control_id_bases.in b/xlat/v4l2_control_id_bases.in
index f3fd9255..a0f9ccae 100644
--- a/xlat/v4l2_control_id_bases.in
+++ b/xlat/v4l2_control_id_bases.in
@@ -1,7 +1,13 @@
 V4L2_CID_BASE				(V4L2_CTRL_CLASS_USER | 0x900)
-V4L2_CID_USER_MEYE_BASE			(V4L2_CID_BASE + 0x1000)
-V4L2_CID_USER_BTTV_BASE			(V4L2_CID_BASE + 0x1010)
-V4L2_CID_USER_TI_VPE_BASE		(V4L2_CID_BASE + 0x1050)
+V4L2_CID_USER_MEYE_BASE			(V4L2_CID_BASE | 0x1000)
+V4L2_CID_USER_BTTV_BASE			(V4L2_CID_BASE | 0x1010)
+V4L2_CID_USER_S2255_BASE		(V4L2_CID_BASE | 0x1030)
+V4L2_CID_USER_SI476X_BASE		(V4L2_CID_BASE | 0x1040)
+V4L2_CID_USER_TI_VPE_BASE		(V4L2_CID_BASE | 0x1050)
+V4L2_CID_USER_SAA7134_BASE		(V4L2_CID_BASE | 0x1060)
+V4L2_CID_USER_ADV7180_BASE		(V4L2_CID_BASE | 0x1070)
+V4L2_CID_USER_TC358743_BASE		(V4L2_CID_BASE | 0x1080)
+V4L2_CID_USER_MAX217X_BASE		(V4L2_CID_BASE | 0x1090)
 #ifndef STRACE_WORKAROUND_FOR_V4L2_CID_USER_IMX_BASE
 # define STRACE_WORKAROUND_FOR_V4L2_CID_USER_IMX_BASE
 /*
@@ -17,7 +23,7 @@ V4L2_CID_USER_TI_VPE_BASE		(V4L2_CID_BASE + 0x1050)
  */
 # undef V4L2_CID_USER_IMX_BASE
 #endif
-V4L2_CID_USER_IMX_BASE			(V4L2_CID_BASE + 0x10b0)
+V4L2_CID_USER_IMX_BASE			(V4L2_CID_BASE | 0x10b0)
 V4L2_CID_MPEG_BASE			(V4L2_CTRL_CLASS_MPEG | 0x900)
 V4L2_CID_MPEG_CX2341X_BASE		(V4L2_CTRL_CLASS_MPEG | 0x1000)
 V4L2_CID_MPEG_MFC51_BASE		(V4L2_CTRL_CLASS_MPEG | 0x1100)
-- 
2.17.1

